src/server/bg_services/buckets_reclaimer.js

```diff
@@ -58,6 +53,23 @@ class BucketsReclaimer {
                 has_errors = true;
             }
         }));
+        if (this.buckets_to_delete.length) {
+            try {
+                await system_store.make_changes({
+                    remove: {
+                        buckets: this.buckets_to_delete,
+                        tieringpolicies: this.tiering_to_delete,
+                        tiers: this.tiers_to_delete,
+                    },
+                    update: {
+                        accounts: this.account_updates,
```

Comment:
@jackyalbo 
If there are no implicit permissions to the `buckets_to_delete`, you will get an empty array and send an empty array to the make_changes.

Reply:
now checking if account_updates is not empty before sending the update